ccx_2.21.c

```diff
@@ -0,0 +1,1923 @@
+/*     CalculiX - A 3-dimensional finite element program                 */
```

Comment:
Comparing this file with `ccx_2.20.c`, it looks like this file does not contain the preCICE-specific changes. Did you maybe accidentally replace the complete file with the unmodified Calculix?